scout/server/blueprints/cases/templates/cases/matchmaker.html

```diff
@@ -0,0 +1,236 @@
+                            <td>{{match_result.node.label}}</td>
+                            <td>{{match_result.patient.contact.name}}
+                              {% if match_result.patient.contact.institution %}
+                                <br>{{match_result.patient.contact.institution}}
+                              {% endif %}
+                              <br>{{match_result.patient.contact.href}}
+                            </td>
+                          </tr>
+                        </tbody>
+                      </table>
+                      <table class="table table-condensed">
+                        <thead>
+                          <tr>
+                            <th scope="col">Diagnoses</th>
+                            <th scope="col">Features</th>
+                            <th scope="col">Genomic Features</th>
+                          </tr>
+                        </thead>
+                        <tbody>
+                          <tr>
+                            <td class="col-md-1">
+                              {% for omim in match_result.patient.disorders %}
+                                <span data-toggle="tooltip" title="{{omim.label or 'description not available'}}" class="label label-info">{{omim.id}}</span>
+                              {% endfor %}
+                            </td>
+                            <td class="col-md-3">
+                              {% for feature in match_result.patient.features %}
+                                <span data-toggle="tooltip" title="{{feature.label or 'description not available'}}" class="label label-primary">{{feature.label}}({{feature.id}})</span>
+                              {% endfor %}
+                            </td>
+                            <td class="col-md-6">
+                              <div class="container-fluid">
+                                <div class="row">
+                                {% for g_feat in match_result.patient.genomicFeatures %}
+                                  <div class="col-lg-4">
+                                    <div class="panel panel-default">
+                                      <div class="panel-body">
+                                        {% for key, value in g_feat.items() %}
+                                          {% if key == 'gene' %}
+                                            {{key}}:&nbsp;<span class="label label-default">{{value.id}}</span>
+                                            <hr>
+                                          {% elif key == 'variant' %}
+                                            {% for ikey, item in value.items() %}
+                                              {% if not ikey == 'shareVariantLevelData' %}
+                                                {{ikey}}:<strong>{{item}}</strong><br>
+                                              {% endif %}
+                                            {% endfor %}
```

Comment:
This one lacks the distinction between 'type' and zygosity, doesn't it? I could not see zygosities of the matching patients. Perhaps either comment up this section: it is a little difficult to follow, or if possible make a subroutine that can be reused?

Reply:
Right, I didn't realize. genomic features are objects like this:

![image](https://user-images.githubusercontent.com/28093618/55476209-434e8780-5616-11e9-952d-551223ff6e4e.png)

I'll fix it!